src/aux-common/bots/BotEvents.ts

```diff
@@ -882,6 +883,157 @@ export interface ImageClassifierOptions {
     cameraType?: CameraType;
 }
 
+export interface OpenClassifyImagesAction
+    extends AsyncAction,
+        ClassifyImagesOptions {
+    type: 'show_classify_images';
+
+    /**
+     * Whether the image classifier should be visible.
+     */
+    open: boolean;
```

Comment:
@TroyceGowdy The `open` property isn't used for `os.classifyImages()`, so it should be removed.